core/src/main/java/org/elasticsearch/index/mapper/TypeFieldMapper.java

```diff
@@ -88,29 +92,12 @@ public MetadataFieldMapper getDefault(MappedFieldType fieldType, ParserContext c
     }
 
     static final class TypeFieldType extends StringFieldType {
-        private boolean fielddata;
```

Comment:
Do we have to deprecate fielddata in 5.x if we're going to do this? I mean, we certainly don't recommend using it in 5.x but maybe we didn't warn about it?